src/types/DSNP.d.ts

```diff
@@ -25,8 +25,8 @@ export interface Message {
  * Broadcast: a public post
  */
 export interface Broadcast extends Message {
```

Comment:
Wait a second. This is a duplicate of https://github.com/LibertyDSNP/sdk-ts/blob/main/src/messages/messages.ts#L22

How do we have two of these.